test/integration/clusterimageset-updater/output/pools/4-5-auto_clusterpool.yaml

```diff
@@ -15,6 +15,7 @@ metadata:
   namespace: ci-cluster-pool
 spec:
   baseDomain: hive.aws.ci.openshift.org
+  hibernationConfig: null
```

Comment:
Maybe add something like the following to the input to ensure it does not get removed in the output?

```
  hibernationConfig:
    resumeTimeout: 20m
```

Reply:
Added